WooCommerce/Classes/ViewRelated/ReusableViews/LabeledTextViewTableViewCell.swift

```diff
@@ -0,0 +1,88 @@
+import UIKit
+import Yosemite
+/// A table view cell that contains a label and a text view.
+///
+final class LabeledTextViewTableViewCell: UITableViewCell {
+    struct ViewModel {
+        var text: String? = nil
+        var productStatus: ProductStatus
+        var placeholder: String? = nil
+        var textViewMinimumHeight: CGFloat? = nil
+        var isScrollEnabled: Bool = true
+        var keyboardType: UIKeyboardType = .default
+        var onNameChange: ((_ text: String) -> Void)? = nil
+        var onTextDidBeginEditing: (() -> Void)? = nil
+        var style: Style = .headline
+
+    }
+    @IBOutlet weak var productLabelHolder: UIView!
+    @IBOutlet weak var productStatusLabel: UILabel!
+    @IBOutlet var productTextField: EnhancedTextView!
+
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        configureLabelStyle()
+        configureBackground()
+    }
+
+    func configure(with viewModel: ViewModel) {
+        productTextField.text = viewModel.text
+        productTextField.placeholder = viewModel.placeholder
+        productStatusLabel.text = viewModel.productStatus.description
+        productTextField.isScrollEnabled = viewModel.isScrollEnabled
+        productTextField.onTextChange = viewModel.onNameChange
+        productTextField.onTextDidBeginEditing = viewModel.onTextDidBeginEditing
+        productTextField.keyboardType = viewModel.keyboardType
+        configureProductStatusLabel(productStatus: viewModel.productStatus)
+    }
+}
+
+private extension LabeledTextViewTableViewCell {
+
+    func configureBackground() {
+        productTextField.backgroundColor = .systemColor(.secondarySystemGroupedBackground)
+    }
+
+
+    func configureLabelStyle() {
+        productStatusLabel.font = UIFont.preferredFont(forTextStyle: .body)
+        productStatusLabel.textAlignment = .center
+        productStatusLabel.textColor = UIColor.black
+        productLabelHolder.backgroundColor = .gray(.shade5)
+        productLabelHolder.layer.cornerRadius = CGFloat(4.0)
+    }
+
+    func configureProductStatusLabel(productStatus: ProductStatus) {
+            if productStatus == ProductStatus.draft {
+                let statusLabel = NSLocalizedString(productStatus.description, comment: "Display label for the product's draft status")
```

Comment:
This won't work for localization, only literal strings are picked up for translation. In this case, since we only show the label for `ProductStatus.draft`, you could go ahead and use `"Draft"` directly.

That said, if the cell can only handle drafts, maybe a `isDraft` flag is more appropriate than passing the full status.